brille/design/Proposed_integrated_syntax_Brill_SpinW_Horace.md

```diff
@@ -0,0 +1,275 @@
+# Workflow examples using Horace, SpinW, and Brille
+**2020-09-22**
+
+## Table of Contents
+
+[Introduction](#introduction)
+
+[Workflow](#workflow)
+
+[Extracting the data](#extracting-the-data)
+
+[Modelling the data](#modelling-the-data)
+
+[Fitting the data with resolution convolution](#fitting-the-data-with-resolution-convolution)
+
+[Implications for implementations](#implications-for-implementations)
+
+## Introduction
+
+[Brille](https://github.com/brille/brille) is a C++ library with Python wrappings for symmetry operations and linear interpolation within an irreducible part of the first Brillouin Zone.
```

Comment:
I thought that Brille did symmetry and interpolation from the whole of q-space ONTO an irreducible part of the BZ